osu.Game/Overlays/BeatmapSet/Scores/NoScoresPlaceholder.cs

```diff
@@ -41,6 +41,10 @@ public void ShowWithScope(BeatmapLeaderboardScope scope)
                 case BeatmapLeaderboardScope.Country:
                     text.Text = BeatmapsetsStrings.ShowScoreboardNoScoresCountry;
                     break;
+
+                case BeatmapLeaderboardScope.Team:
+                    text.Text = BeatmapsetsStrings.ShowScoreboardNoScoresTeam;
+                    break;
```

Comment:
Checking web, web just doesn't show the team scope if you're not on a team... Not sure if this is an issue or not? It definitely may be pretty annoying to match web given how `TabControl` can be a bit jank...

Reply:
Yeah just going with it for now.